File: CPP_Programming/Day_6/program53.cpp
/*  
    Description : Write a program that accepts frequency & prints
                  the below pattern : 
    Input : 4  4
    Output :       *
                 * *
               * * * 
             * * * * 
*/

#include<iostream>
using namespace std;

class Pattern
{
    public :
        int iRow;
        int iCol;

    Pattern(int iRow , int iCol)
    {
        this->iRow = iRow ;
        this->iCol = iCol;
    }

    void Display()
    {
        int i = 0;
        int j = 0;
        
        for(i = 1  ; i <= iRow ; i++)
        {
            for(j = iCol ; j >= 1 ; j--)
            {
                if((i >= j))
                {
                    cout<<"*"<<"\t";
                }
                else
                {
                    cout<<"\t";
                }
            }
            cout<<"\n";
        }
    }

};

int main()
{
    int iRow = 0;
    int iCol = 0;

    cout<<"Enter the number of rows : ";
    cin>>iRow;

    cout<<"Enter the number of columns : ";
    cin>>iCol;

    Pattern pobj(iRow , iCol);

    pobj.Display();


    return 0;
}

File: Python_Programming/Day_6/program53.py
'''
    Description : Write a program that accepts frequency & prints
                  the below pattern : 
    Input : 4  4
    Output :       *
                 * *
               * * * 
             * * * * 
''' 

def Display(Row , Col):
    i = 0
    j = 0

    for i in range(1 , Row+1 , 1):
        for j in range(Col , 0 , -1):
            if(i >= j):
                print("*\t" , end = "")
            else :
                print("\t" , end = "") 
        print()



def main():
    Row = 0 
    Col = 0

    print("Enter the number of rows : ",end = "")
    Row = int(input())

    print("Enter the number of columns : ", end = "")
    Col = int(input())
    
    Display(Row,Col)

if __name__ == "__main__":
    main()
    